docs/data/joy/components/grid/grid.md

```diff
@@ -0,0 +1,154 @@
+
+{{"demo": "SpacingGrid.js", "bg": true}}
+
+### Row & column spacing
+
+The `rowSpacing` and `columnSpacing` props allow for specifying the row and column gaps independently.
+It's similar to the `row-gap` and `column-gap` properties of [CSS Grid](/system/grid/#row-gap-amp-column-gap).
+
+{{"demo": "RowAndColumnSpacing.js", "bg": true}}
+
+### Responsive values
+
+You can switch the props' value based on the active breakpoint.
+
+{{"demo": "ResponsiveGrid.js", "bg": true}}
+
+Responsive values is supported by:
+
+- `columns`
+- `columnSpacing`
+- `direction`
+- `rowSpacing`
+- `spacing`
+- all the [other props](#system-props) of the system
+
+:::warning
+⚠️ When using a responsive `columns` prop, each grid item needs its corresponding breakpoint.
+For instance, this is not working. The grid item misses the value for `md`:
+
+```jsx
+<Grid container columns={{ xs: 4, md: 12 }}>
+  <Grid xs={2} />
+</Grid>
+```
+
+:::
+
+### Auto-layout
+
+The Auto-layout makes the grid items equitably share the available space.
+This also means that you can set the width of one grid item, and the others will automatically resize around it.
+
+{{"demo": "AutoGrid.js", "bg": true}}
+
+### Variable width content
+
+Set one of the size breakpoint props to `"auto"` instead of `true` or a `number` to render a column based on the natural width of its content.
+
+{{"demo": "VariableWidthGrid.js", "bg": true}}
+
+### Columns
+
+You can change the default number of columns (12) with the `columns` prop.
+
+{{"demo": "ColumnsGrid.js", "bg": true}}
+
+### System props
+
+As a CSS utility component, Grid supports all [MUI System properties](/system/properties/).
+You can use them as props directly on the component.
+For instance, a margin-top:
+
+```jsx
+<Grid mt={2}>
+```
+
+## Interactive demo
+
+Below is an interactive demo that lets you explore the visual results of the different settings:
+
+{{"demo": "InteractiveGrid.js", "hideToolbar": true, "bg": true}}
+
+## Limitations
+
+### Negative margin
+
+The spacing between items is implemented with a negative margin. This might lead to unexpected behaviors. For instance, to apply a background color, you need to apply `display: flex;` to the parent.
+
+### direction: column | column-reverse
+
+The `xs`, `sm`, `md`, `lg`, and `xl` props are **not supported** within `direction="column"` and `direction="column-reverse"` containers.
+
+They define the number of grids the component will use for a given breakpoint. They are intended to control **width** using `flex-basis` in `row` containers but they will impact height in `column` containers.
+If used, these props may have undesirable effects on the height of the `Grid` item elements.
+
+## CSS Grid Layout
+
+The `Grid` component is using CSS flexbox internally.
+But as seen below, you can easily use [the system](/system/grid/) and CSS Grid to layout your pages.
+
+{{"demo": "CSSGrid.js", "bg": true}}
+
+## Common examples
+
+### Profile cards
+
+{{"demo": "ProfileCards.js"}}
```

Comment:
I think "Profile cards" is not a good name if you intend the showcase how to center the elements. Also, let's remove the border radius because it looks strange to me.

<img width="801" alt="Screen Shot 2565-12-22 at 10 03 09" src="https://user-images.githubusercontent.com/18292247/209046403-ea72b4e2-0c0f-429f-a2c5-8b2e1139031f.png">

```suggestion
### Centering children

// add some explanation about how to make the children stay on the center.

{{"demo": "ProfileCards.js"}}
```